integration_test/third_party_apps_data/applications/hbase/centos_rhel/install

```diff
@@ -0,0 +1,36 @@
+set -e
+
+sudo yum -y install \
+    java-1.8.0-openjdk java-1.8.0-openjdk-devel wget
+
+sudo mkdir /opt/hbase
+wget http://apache.mirror.gtcomm.net/hbase/stable/hbase-2.4.9-bin.tar.gz
```

Comment:
Chances are a bit higher that users will use the default package manager (e.g. yum, apt-get, zyypper) to install a package instead of a pinned version `2.4.9`. Can we adjust the `install` script to reflect that?

Reply:
Same as with hbase-metrics where there doesn't appear to be a package

Reply:
the [official docs ](https://hbase.apache.org/book.html#quickstart)also point to using a tar

Reply:
I don't see any distro packaging for hbase either.

Reply:
Ah I see. NVM